lib/util.js

```diff
@@ -454,6 +454,30 @@ const util = {
     ]
     util.run('ninja', ninjaOpts, options)
   },
+ 
+  runCoverage: (options = config.defaultOptions) => {
+    console.log('running coverage for ' + config.buildTarget + '...')
+
+    if (process.platform === 'win32') util.updateOmahaMidlFiles()
+    if (process.platform === 'linux') util.prepareWidevineCdmBuild()
+
+    const args = util.buildArgsToString(config.buildArgs())
+    // provide arg for COVERAGE_OUT_DIR
+    util.run('gn', ['gen',
+      'out/brave/coverage', // make an arg - i.e. build with COVERAGE_OUT_DIR as 'out/COVERAGE_OUT_DIR/coverage'
+      '--args="' + args + ' use_clang_coverage=true is_component_build=false dcheck_always_on=true"'], options)
```

Comment:
this is definitely not right, these should always be handled by config.js and you shouldn't be running `gn gen` directly like this anyway, you should be using `util.buildTarget`, but I'm not sure that makes sense here either since I think we should just be running build first?